package.json

```diff
@@ -28,8 +28,8 @@
   "homepage": "https://github.com/auth0/auth0-deploy-cli#readme",
   "dependencies": {
     "auth0": "^2.27.0",
-    "auth0-extension-tools": "^1.5.0",
-    "auth0-source-control-extension-tools": "^4.0.7",
+    "auth0-extension-tools": "^1.4.4",
```

Comment:
Not sure why this went down but I can see in package-lock.json that we remain on 1.5.0.